dht.go

```diff
@@ -108,10 +108,9 @@ type IpfsDHT struct {
 	// networks).
 	enableProviders, enableValues bool
 
-	// maxLastSuccessfulOutboundThreshold is the max threshold/upper limit for the value of "lastSuccessfulOutboundQuery"
-	// of the peer in the bucket above which we will evict it to make place for a new peer if the bucket
-	// is full
-	maxLastSuccessfulOutboundThreshold time.Duration
+	// maxLastSuccessfulOutboundThreshold is the max threshold/upper limit on the time duration
+	// between the current time and the last time a peer was useful to us.
+	maxLastSuccessfulOutboundThreshold float64
```

Comment:
why make this a float? it's only used here to compare with a time duration?

Reply:
Because it's calculated using a math formula that translates to a float64. Take a look at:
https://github.com/libp2p/go-libp2p-kad-dht/blob/82c233366ce60224d2f8741659f0e2143cf47a25/dht.go#L291